src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -473,10 +476,36 @@ export class PageControllerBase {
 
       viewModel.backLink = this.getBackLink(progress)
 
+      const notificationEmailWarning = await this.buildMissingEmailWarningModel(
+        request,
+        isStartPage
+      )
+
+      viewModel.notificationEmailWarning = notificationEmailWarning
```

Comment:
ditto for my earlier comment, you've already done this for the summary page controller 🙂

Reply:
I thought to do it but didn't do it not to put all that in 1 line but ok :)